builder/resubmit_utils_test.go

```diff
@@ -43,7 +43,7 @@ func TestResubmitUtils(t *testing.T) {
 			subAll = append(subAll, submission{time.Now(), subBest})
 			subLast = subBest
 		}
-	})
+	}, nil)
```

Comment:
Could also be equivalent to `time.Now()`

Reply:
updated